lib/rubocop/cop/style/line_end_concatenation.rb

```diff
@@ -36,6 +36,10 @@ def investigate(processed_source)
 
         def autocorrect(operator_range)
           @corrections << lambda do |corrector|
+            # Include any trailing whitespace so we don't create a syntax error.
+            operator_range = range_with_surrounding_space(operator_range,
+                                                          :right, nil,
+                                                          !:with_newline)
```

Comment:
:smile: I think we've discussed the "negated symbol as boolean" idiom before, although I don't remember the outcome of that discussion. There is [one place](https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/style/semicolon.rb#L35) in the code base where we use it already. The idea is that "not with newline" says more than just `false`. Alternative way to express this would be 1) writing `false`and a comment saying that it's not including newline, or 2) having a separate method for not including newline.